src/parser/parsers/heroes.py

```diff
@@ -26,6 +26,11 @@ def run(self):
                     'IsDisabled': hero_value['m_bDisabled'],
                 }
 
+                # Key is missing from released heroes
+                # Frontend will need to use if "m_bAvailableInHeroLabs or not m_bInDevelopment"
+                if 'm_bAvailableInHeroLabs' in hero_value:
+                    hero_stats['InHeroLabs'] = hero_value['m_bAvailableInHeroLabs']
```

Comment:
similar to `InDevelopment` flag, can we instead set the flag, but default to False. ie.

```suggestion
                hero_stats['InHeroLabs'] = hero_value.get('m_bAvailableInHeroLabs', False)
```